build/tasks/aria-supported.js

```diff
@@ -84,7 +84,10 @@ module.exports = function(grunt) {
 				const content = getMdContent(
 					`# ARIA Roles and Attributes ${
 						listType === 'all' ? 'available' : listType
-					} in axe-core.`,
+					} in axe-core.\n\n` +
```

Comment:
Can you limit the character width of this string literal to reduce horizontal scrolling? I've worked on teams with standards of 80 or 100 chars.